spec/controllers/account/profile_controller_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+RSpec.describe Account::ProfileController, type: :controller do
+  before :each do
+    @user = create(:user)
+    sign_in @user
+  end
+
+  describe "GET #show" do
+    it "renders the #show view" do
+      get :show, params: { id: @user.id }
```

Comment:
`id:` key for profile page

Reply:
Roman is right 
in browser what URL you see?
/account/profile/edit?

where is ID here? it's not needed